spyder/plugins/editor/extensions/__init__.py

```diff
@@ -13,4 +13,5 @@
 
 from .closebrackets import CloseBracketsExtension
 from .closequotes import CloseQuotesExtension
+from .docstring import WriterDocstring
```

Comment:
`WriterDocstring` would be more recognizable as an extension if it was called `DocstringWriterExtension`

Reply:
I renamed the classname.